src/components/dialog-full-screen/full-screen-heading/full-screen-heading-classic.style.js

```diff
@@ -0,0 +1,17 @@
+import { css } from 'styled-components';
+import { THEMES } from '../../../style/themes';
+
+export default ({ theme }) => theme.name === THEMES.classic && css`
+  border-bottom-color: #CCD6DA;
+
+  .carbon-app-wrapper {
```

Comment:
Not sure whether this is out of scope for this ticket, but should this be a styled-component?

Reply:
Yes it is out of scope for this ticket.